website/docs/reference/change-requests.md

```diff
@@ -71,7 +72,7 @@ From here, if you have the correct permissions, you can approve and apply the ch
 
 Change requests have their own set of environment-specific permissions that can be applied to [custom project roles](rbac.md#custom-project-roles). These permissions let users
 
-- approve change requests
+- approve/reject change requests
```

Comment:
Are these different permissions? If so, we might want to split them up into two lines here, like this:

```suggestion
- approve change requests
- reject change requests
```

Reply:
It's the same permission and it's updated to Approve/Reject change requests in DB